scene/resources/bit_map.cpp

```diff
@@ -498,7 +498,7 @@ Vector<Vector<Vector2>> BitMap::clip_opaque_to_polygons(const Rect2 &p_rect, flo
 
 				Vector<Vector2> polygon = _march_square(r, Point2i(j, i));
 				print_verbose("BitMap: Pre reduce: " + itos(polygon.size()));
-				polygon = reduce(polygon, r, p_epsilon);
+				polygon = reduce(polygon, r, p_tolerance);
```

Comment:
This is also suspicious. In `reduce` the parameter name is currently called `epsilon`, and then this does some weird math and passes it as an `optimization` parameter. It doesn't look like a comparison epsilon/tolerance.